src/contrib/cluster/Akka.DistributedData.LightningDB/LmdbDurableStore.cs

```diff
@@ -84,6 +83,12 @@ public LmdbDurableStore(Config config)
 
             _path = _config.GetString("dir");
 
+            if (string.IsNullOrEmpty(_path))
```

Comment:
Do quick `null` check and the log the warning so the user knows that there's something off with their configuration, but then make the system work using its normal defaults afterwards